### PR TITLE
Add Django 3.2 and 4.0 to the test matrix

### PR DESCRIPTION
**Pull request by jaap3**

### CODE REVIEW

**Comment by jaap3** [comment]
@vkurup Not sure why GitHub is not triggering actions for this PR

**Comment by vkurup** [comment]
Hi @jaap3, I'm not at Caktus anymore, so I can't troubleshoot. Hopefully someone can step in and nudge the CI.

**Comment by jaap3** [comment]
@vkurup ah, sorry for the noise. Hopefully this project is still on the radar for someone at Caktus (@tobiasmcnulty?). It's in a good state and just needs light maintenance to keep up with Django releases.

**Comment by tobiasmcnulty** [comment]
Hi @jaap3, thanks for the PR!

To help troubleshoot, could you possibly push another commit to your branch? I think the workflow run probably required approval, but we missed the 30 day window before it expired: https://docs.github.com/en/actions/managing-workflow-runs/approving-workflow-runs-from-public-forks

**Comment by jaap3** [comment]
The "Add Django 4.0 to the test matrix" commit was actually made yesterday in an attempt to trigger the workflow. So that's not it :-(

I'll try closing this PR and re-opening it. See if that works.

File: tox.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py36,py37,py38,py39}-{2.2,3.0,3.1}
+envlist = {py36,py37,py38,py39}-{2.2,3.0,3.1,3.2,4.0}
```
Comment (tobiasmcnulty) [inline tox.ini:2]: It looks like Django 4.0 officially supports only Python 3.8+; could you change this line to the following?
```
envlist = py3{6,7,8,9}-{2.2,3.0,3.1,3.2},py3{8,9}-{4.0}
```

File: tox.ini [hunk 2]
```diff
@@ -14,3 +14,5 @@ deps =
     2.2: Django>=2.2,<3.0
     3.0: Django>=3.0,<3.1
     3.1: Django>=3.1,<3.2
+    3.2: Django>=3.2,<4.0
+    4.0: Django>=4.0a1,<4.1
```
Comment (tobiasmcnulty) [inline tox.ini:18]: This didn't work for me locally, but changing to `Django>=4.0b1` did.

**Review by tobiasmcnulty — COMMENTED** [review]
🚀  Thanks again for this PR! It works locally with one small tweak (plus one for nitpick for maintainability). I'm still not sure what's going on with Actions, but if you don't mind making these updates I will merge and we'll see what happens on `develop`...

**Pull request closed without merging**